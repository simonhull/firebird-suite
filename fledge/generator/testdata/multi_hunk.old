function alpha() {
	return "alpha";
}

function beta() {
	return "beta";
}

function gamma() {
	return "gamma";
}

function delta() {
	return "delta";
}

function epsilon() {
	return "epsilon";
}

function zeta() {
	return "zeta";
}
