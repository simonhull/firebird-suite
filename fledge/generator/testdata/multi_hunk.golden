--- old.txt
+++ new.txt
@@ -1,3 +1,3 @@
 function alpha() {
-    return "alpha";
+    return "alpha_modified";
 }
@@ -19,5 +19,5 @@
 }
 
 function zeta() {
-    return "zeta";
+    return "zeta_modified";
 }
